File: JS/car.js
class Car
{
    id;
    passenger;
    constructor (license, driver)
    {
        this.license = license;
        this.driver = driver;
    }

    printDataCar()
    {
    console.log(this.driver)
    console.log(this.license)
    console.log(this.driver.name)
    console.log(this.driver.document)

    }

}
/*
Car.prototype.printDataCar = function () {
    console.log(this.driver)
    console.log(this.license)
    console.log(this.driver.name)
    console.log(this.driver.document)
    console.log(this.driver.license)
}
*/

File: Java/Car.java
package Java;

public class Car {
    //atributos
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver )
    {
        this.license = license;
        this.driver = driver; 
    }

    //metodos
    void printDataCar()
    {
        System.out.println("License: " + license + " Name Driver: " + driver.name);
    }
}
